.vib/cilium/goss/goss.yaml

```diff
@@ -64,7 +64,7 @@ file:
     exists: true
     filetype: symlink
   # Cilium CNI configuration files should be present in the host
-  /host{{ .Vars.agent.cniPlugin.hostCNINetDir }}/05-cilium.conflist:
+  /host/etc/cni/net.d/05-cilium.conflist:
```

Comment:
This is the default value (unless specifically overwritten via the `agent.cniPlugin.hostCNINetDir` parameter). I want to ensure the mechanism to set `/etc/cni/net.d` by default when the parameter isn't provided works as expected, hence I proceed to hardcode it.